### PR TITLE
32

### LINKED ISSUE #32: Client/Dockerfile for admin crud
@Andrew-Sem this one is yours

### PR DESCRIPTION
**Pull request by Andrew-Sem**
Closes #32

---

## PR-Codex overview
This PR adds a new environment variable and configures Nginx to serve the app. It also updates dependencies and adds a new `test-prod` script.

### Detailed summary
- Added `VITE_ADMIN_API_URL` environment variable in `docker.env`
- Added Nginx configuration in `qa-admin/src/main/client/nginx.conf`
- Added `test-prod` script in `qa-admin/src/main/client/package.json`
- Updated dependencies in `qa-admin/src/main/client/package.json`
- Added Dockerfile in `qa-admin/src/main/client/Dockerfile`
- Added `rollup` dependency in `node_modules/vite`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: qa-admin/src/main/client/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+ARG VITE_ADMIN_API_URL
```
Comment (l3r8yJ) [inline qa-admin/src/main/client/Dockerfile:11]: remove from here

File: qa-admin/src/main/client/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ENV VITE_ADMIN_API_URL=$VITE_ADMIN_API_URL
```
Comment (l3r8yJ) [inline qa-admin/src/main/client/Dockerfile:12]: and here

**Review by l3r8yJ — CHANGES_REQUESTED** [review]
@Andrew-Sem take a look at my comments below, please 

move this env var [here](https://github.com/l3r8yJ/volpi-qa/blob/master/docker.env)

**Comment by Andrew-Sem** [comment]
@l3r8yJ take a look at this, please

**Review by l3r8yJ — APPROVED** [review]
@Andrew-Sem ltgm

**Pull request merged**